.github/workflows/check_build.yaml

```diff
@@ -0,0 +1,16 @@
+name: Test build Docusaurus source for Pull Requests
+
+on: [push]
+
+jobs:
+  test_build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Install yarn
+        run: npm install yarn
+
+      - name: Install dependencies
+        run: yarn install
```

Comment:
After this step, don't we need to also run `yarn build`?

Reply:
Oh yup. Did I just copy half of this file?? Yikes